src/main/java/net/apnic/rdap/authority/RDAPAuthority.java

```diff
@@ -279,6 +279,16 @@ public static URI normalizeServerURI(URI server)
         return server;
     }
 
+    public void setDefaultServer(URI defaultServerURI)
+    {
+        if(defaultServerURI == null)
+        {
+            throw new IllegalArgumentException("default server uri cannot be null");
```

Comment:
You are correct, Its fine for the internal creation state to be null but doesn't make sense for someone to be explicitly calling this function with a null value. Generally my style to guard against this until I can find a valid reason for allowing this.